app/src/main/java/com/example/warehousemanagement/domain/model/User.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.warehousemanagement.domain.model
+
+data class User(
```

Comment:
Why we dont remove all field like `role`, `picture` into `information` data class nhỉ

Reply:
Done